applications/GeoMechanicsApplication/tests/test_transient_thermal.py

```diff
@@ -0,0 +1,157 @@
+import os
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import test_helper
+
+class KratosGeoMechanicsTransientThermalTests(KratosUnittest.TestCase):
+    """
+    This class contains benchmark tests which are checked with the analytical solution
```

Comment:
Then I would expect the use of something that numerically generates the analytical solution. The type of testing here is regression on a previously obtained value.

Reply:
The comment is updated.